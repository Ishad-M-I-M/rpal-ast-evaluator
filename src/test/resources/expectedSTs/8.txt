gamma
.lambda
..T
..gamma
...Print
...gamma
....gamma
.....gamma
......gamma
.......T
.......A
......B
.....C
....4
.gamma
..Y
..lambda
...T
...lambda
....a
....lambda
.....b
.....lambda
......c
......lambda
.......N
.......gamma
........gamma
.........Conc
.........gamma
..........gamma
...........Conc
...........gamma
............gamma
.............Conc
.............gamma
..............gamma
...............Conc
...............gamma
................gamma
.................Conc
.................gamma
..................gamma
...................Conc
...................->
....................gr
.....................N
.....................1
....................gamma
.....................gamma
......................gamma
.......................gamma
........................T
........................a
.......................c
......................b
.....................-
......................N
......................1
....................
..................Move 
................a
.............. to 
............b
..........\n
........->
.........gr
..........N
..........1
.........gamma
..........gamma
...........gamma
............gamma
.............T
.............c
............b
...........a
..........-
...........N
...........1
.........
