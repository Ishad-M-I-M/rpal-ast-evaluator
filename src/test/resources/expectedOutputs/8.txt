Move C to B\nMove A to B\nMove C to B\nMove C to B\nMove C to B\nMove A to B\nMove C to B\nMove A to B\nMove C to B\nMove A to B\nMove C to B\nMove C to B\nMove C to B\nMove A to B\nMove C to B\n